src/components/Screens.tsx

```diff
@@ -0,0 +1,17 @@
+import styled from "@emotion/styled";
+
+export const XlargeWrapper = styled.div`
+  width: 1200px;
+`;
+
+export const LargeWrapper = styled.div`
+  width: 992px;
+`;
+
+export const MediumWrapper = styled.div`
+  width: 768px;
+`;
+
+export const SmallWrapper = styled.div`
+  width: 576px;
+`;
```

Comment:
we don't need anymore this file, so remove it

Reply:
fixed